cases/function/window/test_window_union.yaml

```diff
@@ -1329,9 +1517,6 @@ cases:
           2, 100, 111, 5
           3, 101, 111, 0
           4, 102, 111, 0
-          5, 0,   114, 7
```

Comment:
do not delete rows

Reply:
This expectation is correct. It may be that copying and pasting causes diff display problems